modules/backup-and-restore/pages/configurations.adoc

```diff
@@ -48,7 +48,14 @@ Typically used when operating in a private network or with a non-AWS S3-compatib
 |System.Backup.CompressProcessNumber | Number of concurrent processes for compression during backup.
 
 It's recommended to keep the default value `0`, which means the number of processes used to compress is equal to the number of CPU cores on each node.
-| `0`
+| `8`
```

Comment:
Confirming: The default value is changing?  Note that that change would invalidate the preceding "It's recommended..."

Reply:
Yes as per the task. Will add the engineer/ticket reporter to confirm.

Reply:
@runlilong can you please take a look at this PR and please confirm everything looks good.